tools/istio-iptables/pkg/cmd/run_test.go

```diff
@@ -418,24 +418,24 @@ func TestHandleInboundIpv6RulesWithOutboundPorts(t *testing.T) {
 	iptConfigurator.handleInboundIpv6Rules(ipv6Range, ipv6Range)
 	actual := FormatIptablesCommands(iptConfigurator.iptables.BuildV6())
 	expected := []string{
-		"ip6tables -t nat -N ISTIO_INBOUND",
-		"ip6tables -t nat -N ISTIO_REDIRECT",
-		"ip6tables -t nat -N ISTIO_IN_REDIRECT",
-		"ip6tables -t nat -N ISTIO_OUTPUT",
-		"ip6tables -t nat -A ISTIO_INBOUND -p tcp --dport 15008 -j RETURN",
-		"ip6tables -t nat -A ISTIO_REDIRECT -p tcp -j REDIRECT --to-ports 15001",
-		"ip6tables -t nat -A ISTIO_IN_REDIRECT -p tcp -j REDIRECT --to-ports 15006",
-		"ip6tables -t nat -A OUTPUT -p tcp -j ISTIO_OUTPUT",
-		"ip6tables -t nat -A ISTIO_OUTPUT -o lo -s ::6/128 -j RETURN",
-		"ip6tables -t nat -A ISTIO_OUTPUT -o lo ! -d ::1/128 -m owner --uid-owner 1337 -j ISTIO_IN_REDIRECT",
-		"ip6tables -t nat -A ISTIO_OUTPUT -o lo -m owner ! --uid-owner 1337 -j RETURN",
-		"ip6tables -t nat -A ISTIO_OUTPUT -m owner --uid-owner 1337 -j RETURN",
-		"ip6tables -t nat -A ISTIO_OUTPUT -o lo ! -d ::1/128 -m owner --gid-owner 1337 -j ISTIO_IN_REDIRECT",
-		"ip6tables -t nat -A ISTIO_OUTPUT -o lo -m owner ! --gid-owner 1337 -j RETURN",
-		"ip6tables -t nat -A ISTIO_OUTPUT -m owner --gid-owner 1337 -j RETURN",
-		"ip6tables -t nat -A ISTIO_OUTPUT -d ::1/128 -j RETURN",
-		"ip6tables -t nat -A ISTIO_OUTPUT -p tcp --dport 32000 -j ISTIO_REDIRECT",
-		"ip6tables -t nat -A ISTIO_OUTPUT -p tcp --dport 31000 -j ISTIO_REDIRECT",
+		"ip6tables -w -t nat -N ISTIO_INBOUND",
```

Comment:
why?
Why does it pass upstream and fail with us?

Reply:
I compared the test output and saw that the actual output needs `-w`. not sure what it does.

Reply:
I think it has something to do with the output of the following?
https://github.com/maistra/istio/blob/9c6658fd6b16846d6cff4a525585a1dcb949f2a6/tools/istio-iptables/pkg/cmd/run_test.go#L58

Reply:
This was likely added by https://github.com/maistra/istio/pull/242
@luksa Any input here?

Reply:
I believe that was added to keep it from failing if it was trying to update iptables at the same time as the platform.

Reply:
True. My question was more like, Has this changed upstream between 1.9.5 and 1.9.8? Because those tests are passing currently and failing with the rebase.

Reply:
Looking at these changes, I think the test may have changed.  I'd check the gid and uid that are being set.  Should be easy to see from the test failures.  The only thing we do differently is allow uid and gid to be dynamically specified, as opposed to hard-coded (1337 is the default).

Reply:
We add the `-w`, but upstream does not.